jgrapht-core/src/main/java/org/jgrapht/Graphs.java

```diff
@@ -576,6 +580,26 @@
     {
         return !graph.incomingEdgesOf(vertex).isEmpty();
     }
+
+    /**
+     * Compute a new mapping from the vertices of a graph to the integer range $[0, n)$
+     * where $n$ is the number of vertices in the graph.
+     *
+     * @param graph the input graph
+     * @param <V> the graph vertex type
+     * @param <E> the graph edge type
+     * @throws NullPointerException if {@code graph} is {@code null}
+     *
+     * @return the mapping as a pair consisting of the {@code vertexMap} and the {@code indexList}
+     *
+     * @see VertexToIntegerMapping
+     */
+    public static <V, E> Pair<Map<V, Integer>, List<V>> getVertexToIntegerMapping(Graph<V, E> graph){
```

Comment:
If Java had tuple assignment like scala, returning a Pair would be helpful, but since it doesn't, I think we should just return the VertexToIntegerMapping.